spyder_vim/vim_widget.py

```diff
@@ -294,6 +315,8 @@ def on_text_changed(self, text):
         if text.startswith("0"):
             # Special case to simplify regexp
             repeat, key, leftover = 1, "0", text[1:]
+        elif text.startswith("G"):
+            repeat, key, leftover = 0, "G", text[1:]
```

Comment:
Use `-1` rather than `0`, it will be clearer (-1 as an index in python means the last one).